crates/context/config/src/types.rs

```diff
@@ -160,8 +175,19 @@ impl ApplicationSource<'_> {
     }
 }
 
-#[derive(Eq, Ord, Debug, Default, Clone, PartialEq, PartialOrd)]
-#[near(serializers = [borsh, json])]
+#[derive(
+    Eq,
+    Ord,
+    Debug,
+    Default,
+    Clone,
+    PartialEq,
+    PartialOrd,
+    Serialize,
+    Deserialize,
+    BorshSerialize,
+    BorshDeserialize,
```

Comment:
It would be nice if we could keep our `derive`s in alphabetical order - I went through and very carefully ordered them all a couple of weeks ago 😅

Reply:
I think there's more value to visibility & readability for derives, and as such, I prefer it to have a visual aid in the form of it ascending length-wise, with related traits grouped together.

Reply:
I didn't spot this before as it was closed, but just to say, the ordering by length is something I find highly weird... I am not sure how that helps code clarity at all.